transform.go

```diff
@@ -513,6 +530,8 @@ func (t *Transformer) transformOne(expr Expression) *ExecNode {
 		return t.transformOr(expr)
 	case AndExpr:
 		return t.transformAnd(expr)
+	case NotExpr:
```

Comment:
This is a duplicate case from the merge between the new NotExpr implementation and the one originally in this commit.

Reply:
Removed